includes/fields/class-field.php

```diff
@@ -59,6 +59,13 @@ abstract class Field {
 	 */
 	protected $label;
 
+	/**
+	 * Field default value.
+	 *
+	 * @var mixed
+	 */
+	protected $default;
```

Comment:
If we didn't have this separate parameter for fields before, please check how we resolved this previously because we set the field defaults for sure, maybe it's possible to use 'value' => ... to set defaults? You can check how this was done previously in the Admin component or other base components (e.g. in meta boxes logic), if you can re-use 'value' it's ok to remove these API changes

Reply:
Plz clarify why a new parameter for the default value is required